### PR TITLE
CPAchecker commands removed

### PR DESCRIPTION
**Pull request by ghost**
the esbmcheck.py file, updated to match with the last verison used on sv-comp19

### CODE REVIEW

File: modules/bmc_check/esbmccheck.py [hunk 1]
```diff
@@ -34,30 +36,29 @@ def __init__(self):
...
-        self.esbmc_solver_op = "--z3"
+        self.esbmc_solver_op = "--boolector"
         self.use_counter_example = False
         # k-induction options
         self.esbmc_basecase_op = "--base-case"
         self.esbmc_forwardcond_op = "--forward-condition"
-        self.esbmc_inductivestep_op = "--inductive-step --no-slice --show-counter-example"
+        self.esbmc_inductivestep_op = "--inductive-step "
 
-        self.dldv_error = " -DLDV_ERROR=ERROR "
```
Comment (lucasccordeiro) [inline modules/bmc_check/esbmccheck.py]: Are we sure about this change? I think it is required by ESBMC, no?

File: modules/bmc_check/esbmccheck.py [hunk 2]
```diff
@@ -34,30 +36,29 @@ def __init__(self):
...
-        self.dassert = " -D_Bool=int "
```
Comment (lucasccordeiro) [inline modules/bmc_check/esbmccheck.py]: The same here...

File: modules/bmc_check/esbmccheck.py [hunk 3]
```diff
@@ -34,30 +36,29 @@ def __init__(self):
...
+        self.dldv_error = " "
+        self.dassert = " "
         self.no_bounds_check = ""
         self.no_pointer_check = ""
         self.no_div_by_zero_check = ""
         self.no_assertions = ""
         self.memory_leak_check = ""
         self.no_inductive_step = False
-        self.quiet = " --quiet "
```
Comment (lucasccordeiro) [inline modules/bmc_check/esbmccheck.py]: If you remove this option, then ESBMC will print the entire log file in the output, which might be too big.

File: modules/bmc_check/esbmccheck.py [hunk 4]
```diff
@@ -712,7 +716,7 @@ def hassuccessfulfromesbmc(_esbmcoutput):
 
 
     def checkwithkparallel(self, _cprogrampath, _actual_ce, _listtmpfiles):
-        result_parallel = commands.getoutput(self.esbmcpath + " " + self.esbmc_arch + " " +
```
Comment (lucasccordeiro) [inline modules/bmc_check/esbmccheck.py:715]: Did we remove the architecture information?

File: modules/bmc_check/esbmccheck.py [hunk 5]
```diff
@@ -793,28 +791,26 @@ def kinductioncheck(self, _cprogrampath):
         if self.debug:
             print(">> Starting the verification of the P\' program")
 
-        #try:
-        #    file = open(_cprogrampath, "r")
-        #    lines = file.readlines()
-        #    res = [y for y in lines if "safe_write" in y or "fflush_all" in y or "bb_show_usage" in y or "bb_error_msg_and_die" in y]
```
Comment (lucasccordeiro) [inline modules/bmc_check/esbmccheck.py:799]: Do we now support the busybox benchmarks?

File: modules/bmc_check/esbmccheck.py [hunk 6]
```diff
@@ -929,48 +952,76 @@ def getnumbeginfuncts(_cfilepath):
 
         return listbeginnumfuct
 
-    def configureCPACheckerPath(self):
+    def configureUAPath(self):
+        return "./Ultimate.py " + self.listproperty + " 64bit "
```
Comment (lucasccordeiro) [inline modules/bmc_check/esbmccheck.py:956]: Are we using ultimate?

**Review by lucasccordeiro — CHANGES_REQUESTED** [review]

**Comment by lucasccordeiro** [comment]
@omaralhawi: do you have any news about this?

File: modules/bmc_check/esbmccheck.py (hunk 5 above)
Comment (ghost) [reply modules/bmc_check/esbmccheck.py:799]: Yes.

File: modules/bmc_check/esbmccheck.py (hunk 4 above)
Comment (ghost) [reply modules/bmc_check/esbmccheck.py:715]: Yes, the esbmc architecture was removed from the entire code .

File: modules/bmc_check/esbmccheck.py (hunk 3 above)
Comment (ghost) [reply modules/bmc_check/esbmccheck.py]: The option is set back to the code after tested.

File: modules/bmc_check/esbmccheck.py (hunk 2 above)
Comment (ghost) [reply modules/bmc_check/esbmccheck.py]: The option is set back to the code after tested.

File: modules/bmc_check/esbmccheck.py (hunk 1 above)
Comment (ghost) [reply modules/bmc_check/esbmccheck.py]: The option is set back to the code after tested.

File: modules/bmc_check/esbmccheck.py (hunk 6 above)
Comment (ghost) [reply modules/bmc_check/esbmccheck.py:956]: Depthk used before CPAchecker and uautomizer. The cpachecker removed while we kept uautomizer tool and ultimate.py is the key for it.

**Review by lucasccordeiro — APPROVED** [review]

**Pull request merged**